src/o365/spo/commands/orgassetslibrary/orgassetslibrary-add.ts

```diff
@@ -0,0 +1,140 @@
+import {
+  ContextInfo, ClientSvcResponse, ClientSvcResponseContents
+} from '../../spo';
+import config from '../../../../config';
+import request from '../../../../request';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption,
+  CommandError,
+  CommandValidate
+} from '../../../../Command';
+import SpoCommand from '../../../base/SpoCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  libraryUrl: string;
+  thumbnailUrl?: string;
+  cdnType?: string;
+}
+
+class SpoOrgAssetlibAddCommand extends SpoCommand {
```

Comment:
Following the naming convention, the class should be named `SpoOrgAssetsLibraryAddCommand`